editor/fbx_importer_manager.cpp

```diff
@@ -0,0 +1,157 @@
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "fbx_importer_manager.h"
+#include "editor/editor_scale.h"
+#include "editor/editor_settings.h"
+#include "scene/gui/link_button.h"
+
+void FBXImporterManager::show_dialog(bool p_exclusive) {
+	String fbx2gltf_path = EDITOR_GET("filesystem/import/fbx/fbx2gltf_path");
+	fbx_path->set_text(fbx2gltf_path);
+	_validate_path(fbx2gltf_path);
+
+	set_flag(Window::FLAG_BORDERLESS, p_exclusive); // Avoid closing accidentally .
+	set_close_on_escape(!p_exclusive);
+
+	popup_centered();
+}
+
+void FBXImporterManager::_validate_path(const String &p_path) {
+	String error;
+	bool success = false;
+
+	if (p_path == "") {
+		error = TTR("Path to FBX2gLTF2 executable is empty.");
+	} else if (!FileAccess::exists(p_path)) {
+		error = TTR("Path to FBX2gLTF2 executable is invalid.");
+	} else {
+		List<String> args;
+		args.push_back("--version");
+		int exitcode;
+		Error err = OS::get_singleton()->execute(p_path, args, nullptr, &exitcode);
+
+		if (err == OK && exitcode == 0) {
+			success = true;
+		} else {
+			error = TTR("Error executing this file (wrong version or architecture).");
+		}
+	}
+
+	if (success) {
+		path_status->set_text(TTR("FBX2gLTF2 executable is valid."));
+		path_status->add_theme_color_override("font_color", path_status->get_theme_color(SNAME("success_color"), SNAME("Editor")));
+		get_ok_button()->set_disabled(false);
+	} else {
+		path_status->set_text(error);
+		path_status->add_theme_color_override("font_color", path_status->get_theme_color(SNAME("error_color"), SNAME("Editor")));
+		get_ok_button()->set_disabled(true);
+	}
+}
+
+void FBXImporterManager::_select_file(const String &p_path) {
+	fbx_path->set_text(p_path);
+	_validate_path(p_path);
+}
+
+void FBXImporterManager::_path_confirmed() {
+	String path = fbx_path->get_text();
+	EditorSettings::get_singleton()->set("filesystem/import/fbx/fbx2gltf_path", path);
+	EditorSettings::get_singleton()->save();
+}
+
+void FBXImporterManager::_browse_install() {
+	if (fbx_path->get_text() != String()) {
+		browse_dialog->set_current_file(fbx_path->get_text());
+	}
+
+	browse_dialog->popup_centered_ratio();
+}
+
+FBXImporterManager::FBXImporterManager() {
+	singleton = this;
+
+	set_title(TTR("Configure FBX Importer"));
+
+	VBoxContainer *vb = memnew(VBoxContainer);
+	vb->add_child(memnew(Label(TTR("FBX2gLTF2 is required for importing FBX files.\nPlease download it and provide a valid path to the binary:"))));
+	LinkButton *lb = memnew(LinkButton);
+	lb->set_text(TTR("Follow this link to obtain FBX2gLTF2"));
+	lb->set_url("http://godotengine.org/fbx-import");
```

Comment:
Avoid HTTP -> HTTPS redirect (which takes time):

```suggestion
	lb->set_url("https://godotengine.org/fbx-import");
```